yt/data_objects/grid_patch.py

```diff
@@ -0,0 +1,437 @@
+"""
```

Comment:
Wait are you reintroducing a whole deleted file here ?

Reply:
Whoops, yes, looks like I did that by mistake.